plugin/src/main/java/net/dzikoysk/funnyguilds/feature/placeholders/GuildPlaceholders.java

```diff
@@ -0,0 +1,147 @@
+                        return StringUtils.repeated(lives, config.livesRepeatingSymbol.full.getValue()) +
+                                StringUtils.repeated(config.warLives - lives, config.livesRepeatingSymbol.empty.getValue());
+                    }
+                    else {
+                        return StringUtils.repeated(config.warLives, config.livesRepeatingSymbol.full.getValue()) +
+                                config.livesRepeatingSymbol.more.getValue();
+                    }
+                }, () -> messages.livesNoValue))
+                .property("lives-symbol-all", new GuildPlaceholder(
+                        guild -> StringUtils.repeated(guild.getLives(), config.livesRepeatingSymbol.full.getValue()),
+                        () -> messages.livesNoValue))
+                .property(Arrays.asList("position", "rank"), new GuildRankPlaceholder(
+                        (guild, rank) -> guildRankManager.isRankedGuild(guild)
+                                ? String.valueOf(guild.getRank().getPosition(DefaultTops.GUILD_AVG_POINTS_TOP))
+                                : messages.minMembersToIncludeNoValue,
+                        () -> messages.minMembersToIncludeNoValue))
+                .property("total-points", new GuildRankPlaceholder((guild, rank) -> rank.getPoints(), () -> 0))
+                .property(Arrays.asList("avg-points", "points"), new GuildRankPlaceholder((guild, rank) -> rank.getAveragePoints(), () -> 0))
+                .property("points-format", new GuildRankPlaceholder(
+                        (guild, rank) -> NumberRange.inRangeToString(rank.getAveragePoints(), config.pointsFormat)
+                                .replace("{POINTS}", String.valueOf(guild.getRank().getAveragePoints())),
+                        () -> NumberRange.inRangeToString(0, config.pointsFormat)
+                                .replace("{POINTS}", "0")))
+                .property("kills", new GuildRankPlaceholder((guild, rank) -> rank.getKills(), () -> 0))
+                .property("avg-kills", new GuildRankPlaceholder((guild, rank) -> rank.getAverageKills(), () -> 0))
+                .property("deaths", new GuildRankPlaceholder((guild, rank) -> rank.getDeaths(), () -> 0))
+                .property("avg-deaths", new GuildRankPlaceholder((guild, rank) -> rank.getAverageDeaths(), () -> 0))
+                .property("assists", new GuildRankPlaceholder((guild, rank) -> rank.getAssists(), () -> 0))
+                .property("avg-assists", new GuildRankPlaceholder((guild, rank) -> rank.getAverageAssists(), () -> 0))
+                .property("logouts", new GuildRankPlaceholder((guild, rank) -> rank.getLogouts(), () -> 0))
+                .property("avg-logouts", new GuildRankPlaceholder((guild, rank) -> rank.getAverageLogouts(), () -> 0))
+                .property("kdr", new GuildRankPlaceholder((guild, rank) -> String.format(Locale.US, "%.2f", rank.getKDR()), () -> 0.0))
```

Comment:
to jeszcze bardziej skomplikowane niż było, i się narobiła nieczytelna masa kodu -> `GuildRankPlaceholder`